src/App.tsx

```diff
@@ -1,14 +1,68 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import debounce from 'lodash.debounce';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { PeopleList } from './components/PeopleList/PeopleList';
```

Comment:
<img width="648" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/70220667/a752050c-44ae-499d-ae85-94e9685374b7">
you should add this file to git